corehq/ex-submodules/casexml/apps/phone/data_providers/case/batched.py

```diff
@@ -144,22 +146,22 @@ def __init__(self, restore_state, chunk_size=None):
         self.domain = self.restore_state.domain
 
         try:
-            self.owner_keys = [[owner_id, False] for owner_id in self.restore_state.owner_ids]
+            self.owner_ids = list(self.restore_state.owner_ids)
         except AttributeError:
-            self.owner_keys = [[self.user.user_id, False]]
+            self.owner_ids = [self.user.user_id]
 
-        self.case_sharing = len(self.owner_keys) > 1
+        self.case_sharing = len(self.owner_ids) > 1
         self.global_state = GlobalSyncState(self.last_synclog, self.case_sharing)
         self.batch_count = 0
 
     def batches(self):
-        for key in self.owner_keys:
+        for owner_id in self.owner_ids:
             batch = CaseSyncCouchBatch(
                 self.global_state,
                 self.domain,
                 self.last_synclog,
                 self.chunk_size,
-                key,
+                owner_id,
```

Comment:
isn't this a different value from `key`? from what i understand, `key = [[owner_id, False]]` is that equivalent to just passing the `owner_id`?